packages/front-end/components/Experiment/ResultsTable.tsx

```diff
@@ -107,6 +106,10 @@ const ROW_HEIGHT = 56;
 const METRIC_LABEL_ROW_HEIGHT = 44;
 const SPACER_ROW_HEIGHT = 6;
 
+export enum RowError {
+  QUANTILE_AGGREGATION_ERROR = "Quantile metrics cannot be re-aggregated across pre-computed dimension breakdowns.",
```

Comment:
question: Should we just have the identifier in the enum like `QUANTILE_AGGREGATION_ERROR = 'QUANTILE_AGGREGATION_ERROR'`? Then we can define either in shared (if needed for both back-end and front-end)?

Right now the object with this error would have the full string instead of just an indentifier.

I think it is fine for now if we are only dynamically constructing this error, but if we ever save this somewhere then updating the string would be harder and we would need to keep multiple versions around.

Reply:
Yeah, no need for the text, changed it to an identifier. The string can go somewhere shared if and when we need to.